### PR TITLE
Victor/cartpole concepts

### PR DESCRIPTION
**Pull request by shnayder**
Add a simple multi-concept example.

Also:
- iteration limit param for testing
- normalize state angles to -pi..pi
- click to change target position in local renderer

### CODE REVIEW

File: Python/samples/cartpole/multi-concept.ink
Comment (ForrestTrepte) [inline Python/samples/cartpole/multi-concept.ink:88]: Was it intended to reference a sim package with the name "RefactoredCartpole" or is a remnant of the name that you happened to use in your testing?

File: Python/samples/cartpole/multi-concept.ink
Comment (ForrestTrepte) [inline Python/samples/cartpole/multi-concept.ink:94]: Why is the goal expressed in this way? Is seems like minimizing RangeBelow(0) wouldn't actually be minimizing anything when the absolute value can't go negative. Am I interpreting this correctly? If this is a fancy way to get desirable system behavior, perhaps a comment to explain?

File: Python/samples/cartpole/multi-concept.ink
Comment (ForrestTrepte) [inline Python/samples/cartpole/multi-concept.ink:1]: Would an overall comment about the purpose of this example and the principle that it is teaching be helpful? If someone is new to this, should they be looking at machine_teacher.ink or multi-concept.ink?

Comment (ForrestTrepte) [reply Python/samples/cartpole/multi-concept.ink:1]: Maybe summarize the roles of ComputeDelta, GoLeft, GoRight, and PickOne?

**Review by ForrestTrepte — APPROVED** [review]
LGTM overall. I left some suggestions of a couple things that might benefit from clarification.

File: Python/samples/cartpole/sim/render.py
Comment (nmann4) [inline Python/samples/cartpole/sim/render.py:132]: Should we remove these for the sample? They seem like debug logs.

**Review by nmann4 — APPROVED** [review]
lgtm

File: Python/samples/cartpole/multi-concept.ink
Comment (shnayder) [reply Python/samples/cartpole/multi-concept.ink]: Fixed

File: Python/samples/cartpole/multi-concept.ink
Comment (shnayder) [reply Python/samples/cartpole/multi-concept.ink:114]: added a comment. Min/max are a bit awkward in terms of target ranges...

File: Python/samples/cartpole/multi-concept.ink
Comment (shnayder) [reply Python/samples/cartpole/multi-concept.ink]: Good idea. Added!

File: Python/samples/cartpole/sim/render.py
Comment (shnayder) [reply Python/samples/cartpole/sim/render.py]: removed

**Comment by shnayder** [comment]
Thanks for the suggestions @ForrestTrepte and @nmann4. Will wait for @akzaidi to review before merging.

**Review by akzaidi — APPROVED** [review]

**Pull request merged**
